aries-site/src/data/color.js

```diff
@@ -23,7 +23,7 @@ const coreColors = [
     hex: colors['purple!'],
   },
   {
-    name: 'teal',
+    name: 'peal',
```

Comment:
```suggestion
    name: 'teal',
```
I think this just got accidentally changed?

Reply:
yup my fat fingers lol